test/controllers/HomepageControllerSpec.scala

```diff
@@ -71,7 +71,7 @@ class HomepageControllerSpec extends FrontEndTestHelper {
         getAuthorisedSecurityComponents,
         getValidJudgmentUserKeycloakConfiguration,
         consignmentService)
-      val homepagePage = controller.homepage().apply(FakeRequest(GET, "/homepage").withCSRFToken)
+      val homepagePage = controller.judgmentHomepage().apply(FakeRequest(GET, "/judgment/homepage").withCSRFToken)
       status(homepagePage) mustBe OK
```

Comment:
Is it worth adding a test in to check that if you go to '/homepage` as a judgment user, you get redirected to `/judgment/homepage` ?